scripts/checkPriceHistoryForInactiveItems.js

```diff
@@ -0,0 +1,20 @@
+const dynamo = require('../helpers/dynamoInterface');
```

Comment:
deleting this script setInactiveLandtrackerField is more ideal and does this functionality and more, this script doesnt seem too purposeful.